ReleaseProcessAutomation/Steps/PipelineSteps/ContinueReleaseOnMasterStep.cs

```diff
@@ -100,8 +100,7 @@ private void CreateTagAndMerge ()
 
     GitClient.Checkout("master");
 
-    GitClient.MergeBranch(currentBranchName);
-    GitClient.ResolveMergeConflicts();
+    GitClient.MergeBranch(currentBranchName, false, true);
```

Comment:
Did you intentionally not use the named parameters or was this an oversight? If first, I would like to better understand your thoughts on this subject.

Reply:
Just noticed, that in my working directory and the Files changed tab I already have this part with named parameters. WTF GitHub